hi-ml-histopathology/src/histopathology/models/deepmil.py

```diff
@@ -183,23 +183,20 @@ def configure_optimizers(self) -> optim.Optimizer:
     def get_metrics_dict(self, stage: str) -> nn.ModuleDict:
         return getattr(self, f'{stage}_metrics')
 
+    def _compute_bag_labels_logits_and_attn_maps(self, batch: Dict) -> Tuple[Tensor, Tensor, List]:
+        """Compute bag labels, logits and attention maps depending on the nature of batch.
+        Subclasses (TilesDeepMILModule and SlidesDeepMILModule) should specify how this is handled depending on if
+        the tiles are fixed or generated on the fly from whole slide images.
+        """
+        raise NotImplementedError
+
+    def _update_results_with_data_specific_info(self, batch: dict, results: dict) -> None:
```

Comment:
here and in line 186: Methods that are expected to be overridden in subclasses should not be marked private.